draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -1715,167 +1834,175 @@ scope.
+single-domain IP link, supported by the just configured Ethernet
+bundled link, following the same procedure described in
+{{multi-technology-link-setup}} above.
 
 For example, with a reference to {{fig-lag}}, the MDSC can request the
-P-PNC2 to setup a multi-technology single-domain IP Link between IP LTP 5-2 on BR21 and IP LTP 6-2 on P24 supported by the Ethernet bundle link
+P-PNC2 to setup a multi-technology single-domain IP Link between IP LTP
+5-2 on BR21 and IP LTP 6-2 on P24 supported by the Ethernet bundle link
 between ETH LTP 5-1 on BR21 and the Ethernet LTP 6-1 on P24.
 
 ### Multi-technology LAG Update
 
 The P-PNC adds new member(s) to an existing LAG group when the MDSC
-updates at the MPI the configuration of an existing Ethernet bundled
-link adding the multi-technology single-domain Ethernet link(s) being
+updates the configuration of an existing Ethernet bundled link at the
+MPI, adding the multi-technology single-domain Ethernet link(s) being
 created, as described above.
 
 When member links are added or removed from a LAG link, the minimum
-number of active member links required to consider the LAG link as
-being up may also need to be updated.
+number of active member links required to consider the LAG link as up
+may also need to be updated.
 
-For example, with a reference to {{fig-lag}}, the MDSC can request the
-P-PNC2 to add the multi-technology single-domain Ethernet link setup
-between the Ethernet LTP 3-1 on BR21 and the Ethernet LTP 4-1 on P24
-to the existing Ethernet bundle link setup between the Ethernet LTP
-5-1 on node BR21 and the Ethernet LTP 6-1 on node P24.
+For example, with reference to {{fig-lag}}, the MDSC can request
+P-PNC2 to add the multi-technology single-domain Ethernet link set up
+between Ethernet LTP 3-1 on BR21 and Ethernet LTP 4-1 on P24 to the
+existing Ethernet bundle link set up between Ethernet LTP 5-1 on node
+BR21 and Ethernet LTP 6-1 on node P24.
 
-After the LAG configuration has been updated, the P-PNC can also
-update the bandwidth information of the multi-technology single-domain IP
-link supported by the updated Ethernet bundled link.
+After the LAG configuration has been updated, the P-PNC can also update
+the bandwidth information of the multi-technology single-domain IP link
+supported by the updated Ethernet bundled link.
 
 {: #multi-technology-path-properties}
 
 ### Multi-technology TE path properties Configuration
 
 The MDSC can discover the TE path properties (e.g., the list of
-SRLGs, the delay) of a multi-technology IP link from the TE properties of:
+SRLGs, the delay) of a multi-technology IP link from the TE properties
+of:
 
 - the IP LTPs terminating the multi-technology IP link (e.g., the list of
 SRLGs reported by the P-PNC using the packet TE topology model);
 
 - the optical path (e.g., the list of SRLGs reported by the O-PNC
-using the WDM or OTN tunnel model); and
+using the DWDM or OTN tunnel model); and
```

Comment:
I-D.ietf-ccamp-wdm-tunnel-yang defines the 'WDM tunnel' model (and not the 'DWDM tunnel' model)

```suggestion
using the WDM or OTN tunnel model); and
```